src/utils/request.ts

```diff
@@ -41,13 +45,31 @@ const addResponseHeadersAndBody = (body: any, xhr: XMLHttpRequest): IKResponse<U
 export const request = (
     uploadFileXHR: XMLHttpRequest,
     formData: FormData,
+    jwtRequestOptions: JwtRequestOptions,
```

Comment:
This parameter is optional

Reply:
function will always be called with this parameter , iska use karna optional hai

Reply:
Resolved